src/ray/protobuf/gcs_service.proto

```diff
@@ -330,6 +324,8 @@ message CreateActorReply {
   Address actor_address = 2;
   // Info about any refs that the created actor is borrowing.
   repeated ObjectReferenceCount borrowed_refs = 3;
+  // The cause of this actor's death if creation was cancelled.
+  ActorDeathCause death_cause = 4;
```

Comment:
Make it optional?

Reply:
Ah I was going to go with the default enum "NOT_SET" value, but yeah optional would be better.